docs/tutorials/writing-documentation.md

```diff
@@ -102,4 +108,22 @@ BradPitt -> A: Do stuff
 
 ## Deployment
 
-Once your branch is merged into master, a [webhook](https://github.com/thesandboxgame/sandbox-smart-contracts/settings/hooks) is called by Github on Read the Docs to trigger the generation of the documentation.
+Once your branch is merged into master,
+a [webhook](https://github.com/thesandboxgame/sandbox-smart-contracts/settings/hooks) is called by Github on Read the
+Docs to trigger the generation of the documentation.
+
+### Deployment to github pages by hand.
```

Comment:
Is github and readthedocs compatible ? (Can we still run readthedocs after merging this branch ?)

Reply:
I'm using two different things in github:
1) github pages to publish.
2) a github action/workflow to generate the documentation and publish it.
None of them interacts with readthedocs in any way, my understanding is that readthedocs runs his own action and pulls the needed files to generate the docs in a different place.